### DISCUSSION TITLE
Possible to add description responses?

### DISCUSSION THREAD

**Discussion by ruuddeenen** [dedoc/scramble#120] [Q&A] [answered]
Is it possible to add a description to every response. For example 404 responses. Right now only 200 responses automatically get the resource model as descript, but I also want to add a description for other responses. Or default to the http code. This can be helpful for exporting the api documentation.

**Comment by romalytvynenko** [comment]
Hey @ruuddeenen 

Can you show me an example of what you want to get?

**Comment by ruuddeenen** [comment]
Hi @romalytvynenko ,

Sorry for the late reply, I missed the notification.

This is roughly what gets generated as responses for an endpoint;
![image](https://user-images.githubusercontent.com/33053199/232790145-d25f4dd2-51d6-4bc7-8d7f-73f4959df8a7.png)
Which results in the following in (for example) Postman:
![image](https://user-images.githubusercontent.com/33053199/232788255-14584cc5-4dee-46f2-a378-60e08bab6657.png)

I would either like to have a default description for 422 (and other) response codes e.g. `ValidationError with message` OR have the ability to add a custom description to every request using PHPDoc or something.

**Reply by romalytvynenko** [reply, marked as answer]
@ruuddeenen sorry for the late reply.

Thanks for the screenshots, I see.

This is an interesting issue. Postman simply does not correctly use a description of referenced response. Notice how 422 response is a reference. If you check that reference you'll find that there is a description there. This is a description of response that Postman should've shown in the sidebar.

Did you find some workaround for this?

**Reply by ruuddeenen** [reply]
Thanks for the explanation! I did not find an workaround but I accepted it haha.
